app/views/vocab_sheets/show.html.haml

```diff
@@ -0,0 +1,36 @@
+- content_for :scripts do
+  \# javascript_include_tag
+- content_for :head do
+  = stylesheet_link_tag "smoothness/jquery-ui.css"
+
+.vocab-sheet__head
+  .vocab-sheet__controls.vocab-sheet__page-controls
+    = orange_submit_button "vocab_sheet.print", {print: true}
+    = form_tag vocab_sheet_path, method: :delete, class: "" do
+      = orange_submit_button "vocab_sheet.delete"
+    .vocab-sheet__size-controls
+      .vocab-sheet__size-control-box
```

Comment:
I took it out because I was adding the notes view which doesn't have a number, but it totally could be returned to that since it's in a separate container. Doing that now.